sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -591,3 +591,49 @@
   font-family: "BCSans-bold", "BCSans", "Noto Sans", Verdana, Arial, sans-serif;
   color: $brand-blue;
 }
+
+.restrictions-details {
+  font-style: $font-style-normal;
+  font-weight: $font-weight-normal;
+  font-size: $font-size-xsmall;
+  line-height: $line-height-header-xsm;
+}
+
+/* Grid for info with icon to the left */
+.info-grid-icon {
+  grid-area: icon;
+  font-size: 24px;
+  color: $banner-color-info;
+}
+
+.info-grid-header {
+  grid-area: header;
```

Comment:
we also have class for small 
![image](https://user-images.githubusercontent.com/77353155/197045555-706fc883-ae91-4ceb-ac4b-91304979cc18.png) and color class 
![image](https://user-images.githubusercontent.com/77353155/197045735-488aa23c-c8c3-4d22-ada6-6127d944c187.png)

Reply:
ok.Thanks!